### PR TITLE
RadioButton: Add tests

### LINKED ISSUE #292: Tests: RadioButton

### PR DESCRIPTION
**Pull request by westwood846**
Closes #292.

### CODE REVIEW

File: src/Input/RadioButton/RadioButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        name="job-type"
+        value="internship"
+        data-testid="radio-2"
+        onChange={onChange}
+      />
+    </React.Fragment>
+  );
+  return {
+    onChange,
+    container,
```
Comment (rosslo) [inline src/Input/RadioButton/RadioButton.test.tsx]: `container` isn't used, can consider removing it.

**Comment by rosslo** [comment]
@westwood846 Do you think we should add the test cases for `checked` and `theme` props?

File: src/Input/RadioButton/RadioButton.test.tsx (hunk 1 above)
Comment (westwood846) [reply src/Input/RadioButton/RadioButton.test.tsx]: Removed!

**Comment by westwood846** [comment]
> @westwood846 Do you think we should add the test cases for `checked` and `theme` props?

Added a snapshot test for theme. `checked` is actually being tested already (`twoRadioButtonsOneChecked`).

**Comment by rosslo** [comment]
@westwood846 LGTM

**Pull request merged**